core/src/main/java/org/lflang/generator/python/PythonGenerator.java

```diff
@@ -110,9 +108,6 @@ public PythonGenerator(LFGeneratorContext context) {
   private PythonGenerator(
       LFGeneratorContext context, PythonTypes types, CCmakeGenerator cmakeGenerator) {
     super(context, false, types, cmakeGenerator, new PythonDelayBodyGenerator(types));
-    // Add the C target properties because they are used in the C code generator.
-    CompilerProperty.INSTANCE.override(this.targetConfig, "gcc"); // FIXME: why?
```

Comment:
This suggests that the Python target only works with GCC? Let's see if all tests pass...

Reply:
I don't know why it was there. I figured I just try to remove it. Seems to work fine on Mac where clang is the default.

Reply:
:laughing: at the `FIXME` (which is probably mine)